cg/cli/demultiplex/sample_sheet.py

```diff
@@ -29,19 +31,23 @@ def sample_sheet_commands():
 
 @sample_sheet_commands.command(name="validate")
 @click.argument("sheet", type=click.Path(exists=True, dir_okay=False))
+@click.option(
+    "--sheet-type",
+    default=SampleSheetType.NOVASEQ,
+    show_default=True,
+    type=click.Choice(SampleSheetType),
+    help="Instrument sample sheet type",
+)
```

Comment:
This information can be retrieved from the RunParameters.xlm file in a field called `/RfidsInfo/FlowCellMode`. See for example the [fixture run parameters](https://github.com/Clinical-Genomics/cg/blob/master/tests/fixtures/apps/demultiplexing/flowcell-runs/201203_A00689_0200_AHVKJCDRXX/RunParameters.xml) line 26. I was thinking of including in my next PR a function inside the model RunParameters that returns this information and could be used for this purpose, so we could avoid adding more parameters to the CLI command. What do you think? My (still unfinished) PR is #1995, see file `cg/models/demultiplex/run_parameters.py` line 66. I could make it ready for review so we can merge it and make use of this in here

Reply:
Sounds good!I will put this in blocked until #1995 is merged